src/core/platform_installer.mjs

```diff
@@ -234,14 +234,19 @@ export class PlatformInstaller {
     if (!podName) throw new MissingArgumentError('podName is required')
     if (!destPath) throw new MissingArgumentError('destPath is required')
 
+    const recursiveFlag = recursive ? '-R' : ''
     try {
-      const recursiveFlag = recursive ? '-R' : ''
       await this.k8.execContainer(podName, container, `chown ${recursiveFlag} hedera:hedera ${destPath}`)
+    } catch (e) {
```

Comment:
Not sure silencing errors is solid solution.. Maybe create a separate ticket (low priority) to fine tune this by creating a list of files that needs to be changed using `chown` command here.

Reply:
good catch: https://github.com/hashgraph/solo/issues/338